### PR TITLE
Implement Event Notification System Cookbook

### PR DESCRIPTION
**Pull request by SOHAIB-code-png**
This pull request introduces a basic event-driven notification system using the Julep API. The system allows users to define event types, details, and trigger times, and automatically sends notifications when the event requires attention.

- **Agent Setup**: The system sets up an agent to handle event notifications.
- **Event Management**: The system monitors and processes events, such as stock price changes or other user-defined events.
- **Notification Trigger**: Based on predefined conditions, the system triggers notifications to the user.

----

> [!IMPORTANT]
> Introduces an event-driven notification system using Julep API to handle user-defined events and send notifications.
> 
>   - **Event Notification System**:
>     - Implements a basic event-driven notification system using Julep API in `event__notification__system.py`.
>     - Allows users to define event types, details, and trigger times.
>     - Automatically sends notifications when events require attention.
>   - **Agent and Task Setup**:
>     - Sets up an agent with `AGENT_UUID` to handle notifications.
>     - Defines `event_notification_task` with input schema for event type, detail, and trigger time.
>     - Uses `evaluate` step to determine if user notification is needed.
>   - **Functionality**:
>     - `trigger_event_notification()` function triggers notifications based on event inputs.
>     - Example usage provided for stock price change event.
> 
> <sup>This description was created by </sup>[Ellipsis](https://www.ellipsis.dev?ref=julep-ai%2Fjulep&utm_source=github&utm_medium=referral)<sup> for 94d11da61170cd82f95efa33e448bec9cfde0585. It will automatically update as commits are pushed.</sup>

### CODE REVIEW

File: cookbooks/event__notification__system.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import uuid
+import yaml
+import time
+from julep import Client
+
+# Generate unique UUIDs
+AGENT_UUID = uuid.uuid4()
+EVENT_NOTIFICATION_TASK_UUID = uuid.uuid4()
+
+api_key = "eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MzM4NTQzOTUsImlhdCI6MTcyODY3MDM5NSwic3ViIjoiYmIzZjZlMDYtYWE5Yi01OWRkLWJkZmQtOGRmMzk4Y2ZiZjY4In0.SAA0Vi4X68DclecawNvLAihdcsLUaT36oKdGIRunr8AtQPMxWgHSLCub_rPwMmOpV3D0muq5Z--Lw-_br5sqIQ" # Add your Julep API key
```
[the remark this answers is not in the document]
Comment (Vedantsahai18) [reply cookbooks/event__notification__system.py:19]: Please remove your API keys from the commit

File: cookbooks/event__notification__system.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        task_id=EVENT_NOTIFICATION_TASK_UUID,
+        input={
+            "event_type": event_type,
+            "event_detail": {
+                "description": description,
+                "trigger_time": trigger_time
+            }
+        }
+    )
+    time.sleep(2)
```
[the remark this answers is not in the document]
Comment (Vedantsahai18) [reply cookbooks/event__notification__system.py:85]: Also kindly follow the naming convention for cookbooks which is as follow:
<Cookbook No>-<Name of the cookbook>

**Comment by SOHAIB-code-png** [comment]
please merge it

**Comment by Vedantsahai18** [comment]
Please remove your API key from the notebook/.py script

**Pull request closed without merging**